docs/articles/documentation/guides/basic-guides/run-tests.md

```diff
@@ -497,19 +498,21 @@ In this case, tests are distributed across four Safari instances and the same te
 
 > If an uncaught error or unhandled promise rejection occurs on the server during test execution, all tests running concurrently will fail.
 
+### Use Concurrency on Remote Devices
+
 When you run tests on [remote devices](../../reference/command-line-interface.md#remote-browsers),
-create connections for each instance of each browser you test against. When using
-the command line interface, specify this number after the `remote:` keyword. In API, create
-a [browser connection](../../reference/testcafe-api/browserconnection/README.md) for each instance.
+create browser connections for each instance of each browser you test against. The total number of instances is *c\*n*, where `c` is the concurrency factor and `n` is the number of browsers.
+
+In the command line interface, specify the number of browser instances after the `remote:` keyword.  For example, use `-c 2 remote:6` to run tests in Chrome, Firefox, and Edge, with two instances for each browser. If you use the API, create a [browser connection](../../reference/testcafe-api/browserconnection/README.md) for each instance in code.
 
-On a remote device, invoke all the required instances manually. The total number of instances
-should divide by the concurrency parameter `n`. Otherwise, an exception will be thrown.
+Launch all the required browser instances manually on a remote device. The total number of instances
+should divide by the concurrency factor `c`. Otherwise, an exception is thrown.
```

Comment:
You mean it should be divisible by a factor (without a remainder)? I'm not sure `should divide by` is the correct use. I might be wrong though...

```suggestion
should be divisible by the concurrency factor `c`. Otherwise, an exception is thrown.
```

Reply:
Yep, I also wrote "should be divisible" at first but got a correction to "should divide"

Reply:
Please see my explanation below regarding "should be divisible".